doc/api/cli.md

```diff
@@ -336,6 +343,64 @@ file will be created if it does not exist, and will be appended to if it does.
 If an error occurs while attempting to write the warning to the file, the
 warning will be written to stderr instead.
 
+### `--report-directory`
```

Comment:
I would prefer these to be a bit more specific: `--diagnostic-report-directory`, `--diagnostic-report-filename`, etc

Reply:
@jasnell - ok, modified as per the suggestion, thnx.